src/crypto-providers/trezorCryptoProvider.ts

```diff
@@ -129,16 +149,37 @@ const TrezorCryptoProvider: () => Promise<CryptoProvider> = async () => {
     prev_index: input.outputIndex,
   })
 
+  const prepareTokenBundle = (
+    multiAssets: _MultiAsset[],
+  ): TrezorMultiAsset | undefined => {
+    if (multiAssets.length > 0 && !isFeatureSupportedForVersion(TrezorCryptoProviderFeature.MULTI_ASSET)) {
+      throw Error(Errors.TrezorMultiAssetsNotSupported)
+    }
+    const tokenBundle = multiAssets.map(({ policyId, assets }) => {
+      const tokenAmounts = assets.map(({ assetName, coins }) => ({
+        assetNameBytes: assetName.toString('hex'),
+        amount: coins.toString(),
+      }))
+      return {
+        policyId: policyId.toString('hex'),
+        tokenAmounts,
+      }
+    })
+    return tokenBundle.length > 0 ? tokenBundle : undefined
+  }
+
   const prepareChangeOutput = (
-    coins: BigInt,
+    lovelaceAmount: string,
```

Comment:
I'd keep the BigInt interface, see similar comment above for the reasoning behind it

Reply:
Fixed